scripts/test/updateAtlas.js

```diff
@@ -27,8 +27,7 @@ async function main() {
     }
     const atlasArchive = await getLatestAtlasArchive();
     rm("-rf", "tests/testProject/theme");
-    rm("-rf", "tests/testProject/widgets");
```

Comment:
Why this has been removed ?

Reply:
Because in the testProject I need to maintain the datagrid widget and that command is removing this folder and the widget. I updated the unzip command to overwrite then that will create/overwrite the folder/files and keep the widget from the submodule untouched.

Reply:
Oke sounds good 👍